### ISSUE TITLE
Question - Address Standardization

### ISSUE THREAD

**Issue by fwiwDev** [joaocarmo/i18n-postal-address#23]
Thank you for the work on this library!

Is it possible to provide an address string and have it broken up into the respective parts? For example, given:

```
addressString = 'Rua do Pastel, 19, 2700-242 Aveiro, Pará, Brazil'
```

is it possible to get something like this returned:

```
  result: {
    addresss1: 'Rua do Pastel, 19',
    postalCode: '2700-242'
    city: 'Aveiro',
    province: 'Pará',
    country: 'Brazil'
   }
```

Where and international address string is broken into separate attributes?

Thank you!

**Comment by joaocarmo** [comment]
Thank you, @fwiwDev ! Hope you're enjoying the library.

This has been on the [TODO](https://github.com/joaocarmo/i18n-postal-address/blob/master/TODO.md) list for some time. It might be a good idea to start working on it.

Keep an eye out for the next version.

**Comment by joaocarmo** [comment]
@fwiwDev the next version `v0.6.0` with include a form of [experimental string parsing](https://github.com/joaocarmo/i18n-postal-address#experimental-string-parsing) available in node environments only — the web version doesn't support `libpostal`.

**Comment by joaocarmo** [comment]
Solved in #35.
